src/components/FileUpload/index.tsx

```diff
@@ -86,9 +87,10 @@ const FileUpload = (props: TFileUploadProps): JSX.Element | null => {
         <input
           name={name}
           type="file"
+          multiple
```

Comment:
this value should be passed with props

Reply:
we have agreed that the Files Upload component should  work only with an array, no matter whether we uploaded a single file or several files